shopfloor/services/single_pack_transfer.py

```diff
@@ -256,7 +256,14 @@ def _set_destination_and_done(self, package_level, scanned_location):
         # on the move lines
         package_level.location_dest_id = scanned_location
         stock = self._actions_for("stock")
-        stock.validate_moves(package_level.move_line_ids.move_id)
+        package_moves = package_level.move_line_ids.move_id
```

Comment:
@sebalix am I wrong or we have done this N times in many places for the same reason?
Can't we move this to `stock` action?

Reply:
Hum yes, it has been done like this at least in the location content transfer. I'll try to move it to the stock action.

Reply:
I added a commit adding a new method `put_package_level_in_move` in the `stock` action, used by the single pack transfer and location content transfer scenarios.